docs/api-docs/storefront/graphql/graphql-redirects.mdx

```diff
@@ -0,0 +1,368 @@
+**GraphQL Storefront API**
+# Guide to Working with Redirects
+
+BigCommerce's GraphQL Storefront API lets frontend developers modify [Redirects](/docs/rest-content/store-content/redirects##upsert-redirects) from a store. These built-in capabilities also allow developers to customize their hosted or headless storefronts with Redirects.
+
+<Callout type="information">
+  #### Create a Redirect API
+  The [Create a Redirect](/docs/rest-content/store-content/redirects#create-a-redirect) API endpoint is deprecated. Use the [Upsert Redirects](/docs/rest-management/redirects) API endpoint instead to update new redirect data.
+</Callout>
+
+The GraphQL Storefront API lets you set the following redirect attributes and more:
```

Comment:
`you set`
You are not really "setting" anything with the GQL API. You are able to discover if a redirect exists for a certain URL path and follow it to retrieve the destination content if you want.